path.go

```diff
@@ -7,6 +7,7 @@ import (
 	"math"
 
 	"github.com/paulmach/go.geojson"
+	"log"
```

Comment:
restarted the travis build, now it says `./path.go:10: imported and not used: "log"`